.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: objective-c
+xcode_sdk: iphonesimulator  # specify to avoid signing issues
+env:
+    - PROJECT=YLTableView.xcodeproj
+    - BUILD_SCHEME=YLTableView
+    - TEST_SCHEME=YLTableViewTests
```

Comment:
These variables need to be under a `global` key: https://github.com/Yelp/YLTableView/pull/10/files

Otherwise it'll matrix this and split it into 3 separate builds, one for each environment variable.